### ISSUE TITLE
Url Issus no picture

### ISSUE THREAD

**Issue by MrPiil** [AdyRock/com.onvif#25]
Hej I have send you a log from my onvif camera it continue say invalid url. i cant no matter what i try get picture from my camere. i must say i am not programmer, but normal user .

**Comment by AdyRock** [comment]
Hi, sorry it's taken so long to get back to you but I still haven't received the log.
Could you try sending it again.

**Comment by MrPiil** [comment]
Hi. I have changed, router. But i still have the same issue. I have tryed to send a new log file, i hope you receive it. I have tried to get it working for some time now. It is 2 herospeed ipc onvif s profile camera.

**Comment by MrPiil** [comment]
Hi again.  I have installeret, nederst ver of onvif, that i know of. Same on homey software. I have, reset My camera. And connectet Them on homey. No problem software finde Them. But they continues  same URL invalid. Here is latest log.
 Log ID: 2022-01-28T12:01:56.102Z
App version 2.0.5

12:1:56.102: SnapShot error (192.168.0.2): Invalid URL: http://:80/snap.jpg
12:1:56.105: SnapShot error (192.168.0.2): Invalid URL: http://:80/snap.jpg
12:1:56.108: SnapShot error (192.168.0.2): Invalid URL: http://:80/snap.jpg
12:1:56.109: Fetch NOW error (192.168.0.2): Invalid URL: http://:80/snap.jpg

i dont know where to look for solution.if Any.
Morten

**Comment by AdyRock** [comment]
Oh, that URL is not good.
Could you, remove the camera from Homey, make sure the LOG LEVEL settings on the app log screen is set to Basic Information and then try adding the camera again. Once it has been added could you send me the log.
Although I don't seem to have got the pervious logs from you so maybe paste it into the reply.

**Comment by AdyRock** [comment]
Maybe try installing the test version of the app as well as that has better logging.
https://homey.app/en-gb/app/com.onvif/ONVIF-Camera/test/

**Comment by AdyRock** [comment]
Sorry, the info is not coming up. Might have to set the log level to Detailed Information.
You could also try setting the snapshot URL manually and add the device ip to the reported URL (http://192.168.0.2:80/snap.jpg or use 192.168.0.10 whichever camera ait is).
You can change the setting in the Homey device by tapping on the cog in top right corner and then tap on Advanced Settings. Then set the User Snapshot URL.

**Comment by MrPiil** [comment]
Yes. I think it is vorking. I now have picture,  in SnapShot. Not motion. I haven tryed if I get motion detention, Will try it later.

**Comment by MrPiil** [comment]
I now have picture on snapshot. And i still get a message invalid url. But only when i open the app. I can live with that. No pictue i motion event. I dont know if that is possible. And i dont get any motion detection.
